flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/InternalTimeServiceManagerImpl.java

```diff
@@ -93,17 +93,17 @@ private InternalTimeServiceManagerImpl(
      * <p><b>IMPORTANT:</b> Keep in sync with {@link InternalTimeServiceManager.Provider}.
      */
     public static <K> InternalTimeServiceManagerImpl<K> create(
-            CheckpointableKeyedStateBackend<K> keyedStatedBackend,
+            CheckpointableKeyedStateBackend<K> keyedStateBackend,
```

Comment:
I could find many nice cleanups like this but a little bit distracted for review. Maybe a separate commit for such cleanups next time (smile)